src/themer/src/createRowAppearance.js

```diff
@@ -10,7 +10,8 @@ const baseStyle = {
   '&[data-isselectable="true"]': {
     cursor: 'pointer'
   },
-  outline: 'none'
+  outline: 'none',
+  textDecoration: 'none'
```

Comment:
I think this will cover part of it – but it won't attend to `color` which changes based on things like `a:visited` for links.

Reply:
Got it, didn't think of that, will make the change tonight

Reply:
<img width="324" alt="Screen Shot 2020-02-04 at 17 32 56" src="https://user-images.githubusercontent.com/13311268/73803087-8fa5af00-4774-11ea-8d23-5df1c97f3501.png">

Fixed it but only set the color explicitly on the default, the other types override color.